client/src/App.js

```diff
@@ -2,9 +2,9 @@ import React, { Component } from 'react';
 import { Route } from "react-router-dom";
```

Comment:
You're undoing changes to master ~ make sure these changes aren't a part of your pull request